resources/sass/components/_facets.scss

```diff
@@ -0,0 +1,6 @@
+.facets-widget-checkbox .item-list__checkbox,
+.facets-widget-links .item-list__links,
+.facets-widget-date_range .item-list__date_range {
```

Comment:
this is the only thing I had to touch to style this date range widget, so as long as we dont have visual regression there is no point in adding the module to composer or tests